### PR TITLE
chore: adding meshplatform to collie-hub

### PR DESCRIPTION
**Pull request by florianow**

### CODE REVIEW

File: kit/azure/meshplatform/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+name: Azure meshPlatform
+summary: |
+  integrates this platform with meshStack as a meshPlatform to enable self-service for our engineering teams.
+compliance:
```
Comment (JohannesRudolph) [inline kit/azure/meshplatform/README.md:5]: this is not the only capability meshStack adds :-D 

we can consider exporting all the tool2block links from CFMM if we want. 

open q: if i have azure and aws etc. and thus multiple platform modules, will i have this capability implemented multiple times? probably yes

File: kit/azure/meshplatform/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - control: cfmm/tenant-management/multi-cloud-tenant-database-integrated-with-lifecycle-management
+    statement: |
+      A central database of all multi-cloud tenants initiates tenant provisioning and deprovisioning processes. The database acts as an authoritative source of tenants and ensures tenant metadata is always up to date.
+---
+
+# Azure meshPlatform
+
+This kit to integrate Azure as a meshPlatform into meshStack instance. With this module, service principals used by meshStack are created with the required permissions. The output of this module is a set of credentials that need to be configured in meshStack as described in [meshcloud public docs](https://docs.meshcloud.io/docs/meshstack.how-to.integrate-meshplatform.html).
```
Comment (JohannesRudolph) [inline kit/azure/meshplatform/README.md:13]: This kit module integrates Azure into meshStack as a platform using the official [terraform-azure-meshplatform](https://github.com/meshcloud/terraform-azure-meshplatform) module. This module sets up service principals and permissions required for meshStack.

File: kit/azure/meshplatform/documentation.tf [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# meshStack Integration
+
+meshStack integration sets up this AAD tenant as a meshPlatform.
+To do this we create AAD tenant-level service principals, allowing meshStack to access data
+and orchestrate Azure platform functionality.
+
+## Replicator Service Principal
+ - Application Client ID: ${module.meshplatform.replicator_service_principal["Application_Client_ID"]}
+ - Enterprise Application Object ID: ${module.meshplatform.replicator_service_principal["Enterprise_Application_Object_ID"]}
+ - Client_Secret:  ${module.meshplatform.replicator_service_principal["Client_Secret"]}
```
Comment (JohannesRudolph) [inline kit/azure/meshplatform/documentation.tf:12]: f: do not put secrets into emitted documentation page!

File: kit/azure/meshplatform/documentation.tf [hunk 4]
```diff
@@ -0,0 +1,20 @@
+output "documentation_md" {
+  value = <<EOF
+# meshStack Integration
+
+meshStack integration sets up this AAD tenant as a meshPlatform.
+To do this we create AAD tenant-level service principals, allowing meshStack to access data
+and orchestrate Azure platform functionality.
+
+## Replicator Service Principal
```
Comment (JohannesRudolph) [inline kit/azure/meshplatform/documentation.tf:9]: a short explanation would help

**Review by JohannesRudolph — APPROVED** [review]

File: kit/azure/meshplatform/documentation.tf (hunk 4 above)
Comment (JohannesRudolph) [reply kit/azure/meshplatform/documentation.tf:9]: @florianow please describe quickly what each principal does. This should be in the product docs already somewhere. 

If I read the documentation of the cloudfoundation (e.g. as a security auditor) I'm not only interested in what is there (the SPN, name etc.) but also especially _why_ it's there and what it's being used for

File: kit/azure/meshplatform/documentation.tf (hunk 3 above)
Comment (florianow) [reply kit/azure/meshplatform/documentation.tf:12]: its not adding secrets :) its add a text how to get the secret if u needing it

**Pull request merged**